sechub-administration/src/main/java/com/daimler/sechub/domain/administration/project/ProjectMetaData.java

```diff
@@ -0,0 +1,72 @@
+package com.daimler.sechub.domain.administration.project;
+
+import java.io.Serializable;
+
+public class ProjectMetaData implements Serializable {
+
+	private static final long serialVersionUID = -721445637539759118L;
+	
+	private String key;
+	private String value;
+	
+	public ProjectMetaData() {
```

Comment:
If its only for JPA (and maybe for some unit testing) I would appreciate if you could make it package private like done at
`ScanAccess`.

Reply:
done